components/eventing-controller/reconciler/backend/reconciler_test.go

```diff
@@ -202,6 +204,18 @@ var _ = AfterSuite(func() {
 })
 
 var _ = Describe("Backend Reconciliation Tests", func() {
+	var ownerReferences *[]metav1.OwnerReference
+
+	When("Creating a controller deployment", func() {
```

Comment:
Do we really need this `When`? Wouldn't it be enough to keep the `It`s that you have added to make sure every time a new publisher proxy deployment is create, it has the correct owner reference? Since the code you have added in the reconciler, anyway will look up the eventing-controller, if the controller is not running, setting the owner reference fails. Right? If so, then this `When` is already implied by the `It`s.

Reply:
Yes, the `When` is more a documentation in this place.